tests/tests_combination.yml

```diff
@@ -315,15 +342,25 @@
       failed_when: not __result.stat.exists
 
     - name: Check the input call with tag={{ __test_tag }}
-      command: /bin/grep ' *input(type="imfile" file="\/var\/log\/inputdirectory\/\*.log" tag="{{ __test_tag }}"' '{{ __test_inputfiles_conf }}'
+      command: "/bin/grep
+        ' *input(type=\"imfile\"
+        file=\"/var/log/inputdirectory/\\*.log\"
+        tag=\"{{ __test_tag }}\"'
+        {{ __test_inputfiles_conf }}"
       changed_when: false
 
-    - name: Create a test log file with a log message in {{ __test_inputfiles_dir }}
-      shell: echo "<167>Dec 26 01:00:00 11.22.33.44 tag msgnum:00000001:26:ABCDEFGHIJKLMNOPQRSTUVWXYZ" >> "{{ __test_inputfiles_dir }}/test.log"
+    - name: "Create a test log file with a log message in
+      {{ __test_inputfiles_dir }}"
+      shell: "echo
+        '<167>Dec 26 01:00:00 11.22.33.44 tag
+        msgnum:00000001:26:ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+        >> {{ __test_inputfiles_dir }}/test.log"
       changed_when: false
 
     - name: Check the fake test log message in {{ __default_system_log }}
-      command: /bin/grep '{{ __test_tag }} .*ABCDEFGHIJKLMNOPQRSTUVWXYZ$' '{{ __default_system_log }}'
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space